diagnostics-with-elk/azuredeploy.json

```diff
@@ -0,0 +1,367 @@
+        "extensionName": "ELKSimple",
+        "location": "[resourceGroup().location]",
+        "imagePublisher": "Canonical",
+        "imageOffer": "UbuntuServer",
+        "OSDiskName": "osdiskforlinuxsimple",
+        "storageAccountType": "Standard_LRS",
+        "publicIPAddressType": "Dynamic",
+        "vmStorageAccountContainerName": "vhds",
+        "vmName": "logstashvm1",
+        "vmSize": "Standard_D1",
+        "virtualNetworkName": "elkvnet2",
+        "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+        "subnetRef": "[concat(resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworkName')), '/subnets/', 'other')]",
+        "esHost":  "10.0.2.100"
+    },
+    "resources": [
+        {
+          "apiVersion": "2015-01-01",
+          "name": "Elasticsearch",
+          "type": "Microsoft.Resources/deployments",
+          "properties": {
+            "mode": "Incremental",
+            "templateLink": {
+              "uri": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/elasticsearch/azuredeploy.json",
+              "contentVersion": "1.0.0.0"
+            },
+            "parameters": {
+              "adminUsername": { "value": "[parameters('adminUsername')]" },
+              "adminPassword": { "value": "[parameters('adminPassword')]" },
+              "virtualNetworkName": { "value": "[variables('virtualNetworkName')]" },
+              "OS": { "value": "ubuntu" },
+              "authenticationType": { "value": "password" },
+              "sshPublicKey": { "value": "" },
+              "loadBalancerType": { "value": "internal" },
+              "jumpbox": { "value": "Yes" },
+              "vmClientNodeCount": { "value": "[parameters('vmClientNodeCount')]" },
+              "vmSizeClientNodes": { "value": "[parameters('vmSizeClientNodes')]" },
+              "vmSizeMasterNodes": { "value": "[parameters('vmSizeMasterNodes')]" },
+              "vmSizeDataNodes": { "value": "[parameters('vmSizeDataNodes')]" },
+              "vmDataNodeCount": { "value": "[parameters('vmDataNodeCount')]" },
+              "esClusterName": { "value": "[parameters('esClusterName')]" },
+              "esVersion": { "value": "[parameters('esVersion')]" },
+              "afs": { "value": "no" },
+              "marvel": { "value": "no" },
+              "marvelCluster": { "value": "no" },
+              "vmSizeMarvelNodes": { "value": "Standard_D2_v2" },
+              "kibana": { "value": "yes" },
+              "sense": { "value": "no" },
+              "jmeterAgent": { "value": "no" },
+              "cloudAzure": { "value": "no" },
+              "cloudAzureStorageAccount": { "value": "" },
+              "cloudAzureStorageKey": { "value": "" },
+              "templateBase": { "value": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master" }
```

Comment:
Possibly make this a variable so it's easier to change all URLs for testing from your own repo.